dlib_face_detection/CMakeLists.txt

```diff
@@ -0,0 +1,110 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(dlib_face_detection)
+
+find_package(catkin REQUIRED COMPONENTS
+  cob_people_detection
+  cob_perception_msgs
+  cob_vision_utils
+  cv_bridge
+  dlib_lib
+  geometry_msgs
+  message_filters
+  message_generation
+  pcl_ros
+  roscpp
+  sensor_msgs
+  std_msgs
+  tf  
+)
+
+find_package(Boost REQUIRED)
+find_package(BZip2)
+find_package(X11 REQUIRED)
+find_package(OpenCV REQUIRED)
+
+
+add_message_files(
+  FILES
+  Rect.msg
+  RectArray.msg
+)
+
+generate_messages(
+  DEPENDENCIES
+  std_msgs
+)
+
+
+set(URL http://dlib.net/files/shape_predictor_68_face_landmarks.dat.bz2)
+set(DOWNLOAD_PATH ${PROJECT_SOURCE_DIR}/data/shape_predictor_68_face_landmarks.dat.bz2)
+set(EXTRACTED_FILE shape_predictor_68_face_landmarks.dat)
+
+if (NOT EXISTS "${DOWNLOAD_PATH}")
+    file(DOWNLOAD "${URL}" "${DOWNLOAD_PATH}")
+endif()
+
+add_custom_command(
+  OUTPUT "${EXTRACTED_FILE}"
+  PRE_LINK
+  COMMAND bunzip2 -dkf ${PROJECT_SOURCE_DIR}/data/shape_predictor_68_face_landmarks.dat.bz2
+  WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/data
+  COMMENT "Unpacking ${DOWNLOAD_PATH}"
+)
+
+catkin_package(
+  INCLUDE_DIRS
+    ../cob_people_detection/common/include
```

Comment:
you cannot have a relative path to another package's include files this in here!
As this package actually only contains executables, you probably can just put `catkin_package()`, i.e. nothing inside the brackets